src/pocketmine/entity/Human.php

```diff
@@ -69,6 +70,14 @@ class Human extends Creature implements ProjectileSource, InventoryHolder{
 	protected $totalXp = 0;
 	protected $xpSeed;
 
+	public function __construct(Level $level, CompoundTag $nbt){
+		parent::__construct($level, $nbt);
+
+		if(strlen($this->skin) < 64 * 32 * 4){
+			throw new \InvalidStateException((new \ReflectionClass($this))->getShortName() . " must have a valid skin set");
+		}
+	}
```

Comment:
Unnecessary check. setSkin already throws the exception in the call in parent constructor delegation (Entity::__construct → Human::initEntity → Human::setSkin).